src/components/templates/ProductTemplate/ProductTemplate/ProductTemplate.tsx

```diff
@@ -7,17 +7,32 @@ import { ProductTemplateHeader } from '../ProductTemplateHeader/ProductTemplateH
 import FlexContainer from '../../../layout/FlexContainer/FlexContainer';
 import FlexRow from '../../../layout/FlexRow/FlexRow';
 import FlexCol from '../../../layout/FlexCol/FlexCol';
+import styled from 'styled-components';
+import grid from '../../../../constants/grid';
 
 export interface ProductTemplateProps {
   children: React.ReactNode;
-  title: string;
+  title?: string;
   subtitle?: string;
   prevStep?: ReactElement | string;
   progress?: Pick<ProgressProps, 'currentStep' | 'totalSteps'>;
   contentWidth?: number;
   onBackPressed?: MouseEventHandler<HTMLAnchorElement>;
 }
 
+function overlayElementAbove(marginTop: number) {
+  return `
+    margin-top: -${marginTop - 24}px; 
+    @media (min-width: ${grid.breakpoints.m}px) {
+      margin-top: -${marginTop}px; 
+    }
```

Comment:
```suggestion
    ${minMedia.desktop`
    ${css`
       margin-top: -${marginTop}px; 
    `}
  `}
```
see https://github.com/zopaUK/react-components/pull/635/files#diff-4695613f7b9d259bdf7507895860c5e413b13315ce9c6b2b699d591f2377c117L21 for an example

Reply:
I need to use the [m grid breakpoint ](https://github.com/zopaUK/react-components/blob/master/src/constants/grid.ts#L14) here in order to coincide with the [spacing styles](https://github.com/zopaUK/react-components/blob/master/src/components/styles/Spacing.ts). This is to adjust for the different padding-top values of the `<ProductTemplateTitleInnerContainer>` [here](https://github.com/zopaUK/react-components/blob/master/src/components/templates/ProductTemplate/ProductTemplateTitle/ProductTemplateTitle.tsx#L48). Would be worth reviewing this if we can implement a single set of breakpoints across the project.